sdk/nodejs/runtime/closure/createClosure.ts

```diff
@@ -695,9 +697,10 @@ function isDefaultFunctionPrototype(func: Function, prototypeProp: any) {
  * specific properties.  If propNames is not provided, or is empty, serialize out all properties.
  */
 function getOrCreateEntry(
-        obj: any, capturedObjectProperties: CapturedPropertyInfo[] | undefined,
+        obj: any, capturedObjectProperties: CapturedPropertyChain[] | undefined,
         context: Context,
-        serialize: (o: any) => boolean): Entry {
+        serialize: (o: any) => boolean,
+        logInfo: boolean | undefined): Entry {
```

Comment:
`logInfo?: boolean`?

Reply:
I like forcing people to have to remember to call it.  `logInfo?:boolean` means it is optional, which means you might forget it.

Reply:
Does typescript not consider `T | undefined` arguments to be optional?